src/components/table-content-wrapper/table-head/table-head.jsx

```diff
@@ -113,6 +120,8 @@ class TableHead extends Component {
                 scope="col"
                 role="columnheader"
                 unstyledChild
+                valign="bottom"
```

Comment:
isn't valign like obsolete and shouldn't be used ?

Reply:
true, this is deprecated. Sorry @Unic0arn  :)

Reply:
To elaborate, you should get the behaviour you're aiming for with css instead:
```
vertical-align: bottom;
```